site/content/rfds/0004-plugin-api.md

```diff
@@ -702,6 +711,63 @@ array. So `$/items/0`, for example, would select into the `items` field
 of the provided object, and then into the first (`0`-th) element of that
 array.
 
+#### Date-time format details
```

Comment:
I would prefer `DateTime` for capitalization and `datetime` for all lowercase references

Reply:
`DateTime` for the type, datetime for the concept.